api/specs.proto

```diff
@@ -77,14 +77,14 @@ message ServiceSpec {
 	// For now, this is a simple struct, but this can include future information
 	// instructing Swarm on how this service should work on the particular
 	// network.
-	message NetworkAttachmentConfig {
+	message DeprecatedNetworkAttachmentConfig {
 		// Target specifies the target network for attachment. This value may be a
 		// network name or identifier. Only identifiers are supported at this time.
 		string target = 1;
 		// Aliases specifies a list of discoverable alternate names for the service on this Target.
 		repeated string aliases = 2;
 	}
-	repeated NetworkAttachmentConfig networks = 7;
+	repeated DeprecatedNetworkAttachmentConfig deprecated_networks = 7;
```

Comment:
Yeah I wasn't sure which way to go so I threw it in here like this and get opinions.